python/lsst/ts/standardscripts/maintel/track_target_and_take_image_gencam.py

```diff
@@ -100,14 +117,34 @@ def get_schema(cls):
         schema_dict["title"] = "TrackTargetAndTakeImageGenCam v1"
         schema_dict["description"] = "Configuration for TrackTargetAndTakeImageGenCam."
 
-        schema_dict["camera_index"] = dict(
-            type="integer",
-            description="SAL Index for the Generic Camera.",
-            minimum=0,
-            maximum=2147483647,
+        # Leaving the option of ``camera_index`` being an integer for
+        # backwards compatibility, assuming it is needed. Otherwise,
+        # we can force it to be an array with, at least, one element.
+        schema_dict["properties"]["generic_camera"] = dict(
+            type="array",
+            description="Information associated with the Generic Cameras",
+            minItems=1,
+            index=dict(
+                type="integer",
+                description="Index of the Generic Camera SAL component.",
+            ),
+            exp_times=dict(
+                type="array",
+                description="Exposure times (seconds) for each camera.",
+                items=dict(
+                    type="number",
+                    minimum=0,
+                    maximum=2147483647,
```

Comment:
I'd remove the maximum value here. Some cameras actually implement a maximum exp_time which will likely be smaller than this. I'd just avoid setting this altogether, just set the minimum.